microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -13,17 +13,17 @@ ms.date: 03/15/2023
 
 
 <!-- ====================================================================== -->
-## Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on Xbox and HoloLens devices
+## Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on HoloLens devices
```

Comment:
HoloLens 2

Reply:
👍 This change was already made in our [other PR already](https://github.com/MicrosoftDocs/edge-developer/pull/2567/files#diff-b1a613b570630c83b92b81d400898b19b594953b198e37f7a41b4c2fd7a834a3)

Reply:
fixed per latest https://github.com/MicrosoftDocs/edge-developer/pull/2568/files